### ISSUE TITLE
Tooltips cannot be dismissed without moving the pointer or focus

### ISSUE THREAD

**Issue by karlgroves** [WordPress/gutenberg#15344]
Tooltips cannot be dismissed without moving the pointer or focus

-   **Severity:** Low
-   **Affected Populations:**
    -   Low-Vision
    -   Motor Impaired
    -   Cognitively Impaired
-   **Platform(s):**
    -   All / Universal
-   **Components affected:**
    -   Add Block
    -   Block Editing
    -   Block Options
    -   Block Panel
    -   Block Types
    -   Editor Top Bar
    -   Structure and Navigation

#### Issue description

Users who trigger tooltips in the components listed, using pointer hover
or keyboard focus, cannot dismiss the tooltip without moving their
pointer or keyboard focus.

Users of high browser zoom or magnification software may only see a
small area of the page at any one time. Content which appears on hover
or focus may cover the content underneath, but moving the pointer or
focus (in order to dismiss the tooltip) also moves the visible area.

#### Remediation Guidance

Add a method for users to hide the tooltips without moving the pointer
or keyboard focus. The simplest way to achieve this is to add an Escape
key handler which closes the currently-visible tooltip.

#### Relevant standards

-   [1.4.13 Content on Hover or Focus (Level
    AA)](https://www.w3.org/TR/WCAG20/#visual-audio-contrast-without-color)

**Note:** This issue may be a duplicate with other existing accessibility-related bugs in this project. This issue comes from the Gutenberg accessibility audit, performed by [Tenon](https://www.tenon.io) and funded by [WP Campus](https://wpcampus.org/). This issue is GUT-6 in Tenon's report

**Comment by afercia** [comment]
There's already an issue about this, see #15145 and a previous attempt to fix it in this (now closed) PR: #8147

**Comment by melchoyce** [comment]
Thanks @karlgroves! Closing as duplicate.
